xdocs/changes.xml

```diff
@@ -123,21 +123,39 @@ Summary
   <li><pr>725</pr>Add Chinese Simplified Translation for Open Model Thread Group</li>
   <li><pr>5710</pr>Add GitHub Issue templates</li>
   <li><pr>5910</pr>Use Caffeine for caching customizers in TestBeanGUI instead of commons-collections4 LRUMap</li>
-  <li><pr>5713</pr>Update Spock to 2.2-groovy-3.0 (from 2.1-groovy-3.0)</li>
+  <li><pr>5713</pr>Update Spock to 2.3-groovy-3.0 (from 2.1-groovy-3.0)</li>
```

Comment:
I'm not sure what do we do about changelog entries that update the same dependency several times.
For instance, we have updated spock from 2.1 to 2.2, and then to 2.3 in different PRs (commits).
Do we really want to reflect all those updates individually?
I would probably be time-consuming, and it would be unreadable at the same time.

Any thoughts on that?

Reply:
No need to put all history of changes between 2 official releases, just the old version to the new version at the moment of the new release.